ambari-server/src/main/resources/common-services/AMBARI_INFRA_SOLR/0.1.0/package/scripts/command_commons.py

```diff
@@ -352,3 +352,21 @@ def check_folder_exists(dir):
   if returncode:
     return False
   return True
+
+def check_folder_until_size_not_changes(dir):
+  """
+  Call du -d 0 <folder> | cut -f 1 on specific directory until the size not changes (so copy operation has finished)
+  """
+  cmd = format("du -d 0 {dir} | cut -f 1")
+  size_changed = True
+  size_str = "-1"
+  while size_changed:
+    returncode, stdout = call(cmd, user=params.infra_solr_user, timeout=300)
+    if stdout:
+      actual_size_str = stdout.strip()
+      if actual_size_str == size_str:
+        break
```

Comment:
`size_changed` is never set to `False`, so the variable might be removed (or `break` replaced with `size_changed = False`).